### PR TITLE
feat(Rows,Cards): change title font weight from light to regular

### PR DESCRIPTION
**Pull request by yceballost**
We changed titles weight from Light to Regular in certain components:
- Rowlist
- Cards
- Highligted card
- Emtpy state card

Ticket: https://jira.tid.es/browse/WEB-235

### CODE REVIEW

File: src/card.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const CardContent: React.FC<CardContentProps> = ({
                                         </Text1>
                                     </Box>
                                 )}
-                                <Text4 as="h1" light>
```
Comment (pladaria) [inline src/card.tsx:70]: you must re-generate affected screenshot tests

from the repo root:
    yarn test-acceptance -u

then commit and push the result

**Review by pladaria — CHANGES_REQUESTED** [review]
please, regenerate screenshot tests

from the repo root:
```
yarn test-acceptance -u
```
then commit and push the result

**Comment by yceballost** [comment]
> please, regenerate screenshot tests
> 
> from the repo root:
> 
> first make sure you're updated with master your deps are up to date:
> 
> ```
> git fetch
> git merge origin/master
> (commit / push)
> yarn
> ```
> 
> then run the tests updating the snapshots:
> 
> ```
> yarn test-acceptance -u
> ```
> 
> then commit and push the result

I'm getting this error

image

**Comment by pladaria** [comment]
> I'm getting this error

You need to install docker and docker compose

If you don't want to, I can update the screenshots for you

**Comment by yceballost** [comment]
> install docker and docker compose

I already done!

**Review by pladaria — APPROVED** [review]
Good job!

Remember to disable docker while you're not using it, otherwise it will waste your mac resources.

**Review by atabel — APPROVED** [review]
Not sure why the placeholder border color changed :/
image

**Pull request merged**